src/main/java/com/box/sdk/BoxDeveloperEditionAPIConnection.java

```diff
@@ -226,7 +233,24 @@ String jwtConstructAssertion() {
         return assertion;
     }
 
-    PrivateKey decryptPrivateKey() {
+    private String getAlgorithmIdentifier() {
```

Comment:
The actual strings are defined in BouncyCastle dependencies. Returning it from enum itself will mean that either we use the BouncyCastle dependencies in the enum class OR we re-define the strings in the enum. None of them seemed really great to me hence I implemented this helper private method. Let me know your thoughts and we will take it forward from there.